assets/theme-css/styles.css

```diff
@@ -130,13 +129,13 @@ svg.icon {
 }
 
 .chroma {
-  padding: 10px;
+  padding: 0.625rem;
 }
 
 /* reboot stuff needed for pydata-sphinx-theme */
 
 pre {
-  font-size: 0.875em;
+  font-size: 0.875rem;
```

Comment:
I am tempted to remove this. I prefer to keep the code the same size as the normal text.

Reply:
I presume this is to calibrate the regular font wise against code. They should perceptual ly be the same size. The adjustment here feels too large for that.

Reply:
I removed the changes.